scripts/globals/spells/songs/fowl_aubade.lua

```diff
@@ -11,7 +11,7 @@ spell_object.onMagicCastingCheck = function(caster, target, spell)
 end
 
 spell_object.onSpellCast = function(caster, target, spell)
-    return xi.spells.spell_song_enhancing.useEnhancingSong(caster, target, spell)
+    return xi.spells.spell_enhancing_song.useEnhancingSong(caster, target, spell)
```

Comment:
Similar to the naming convention idea for sub-directories.  The global table and function name seem to be a bit redundant.  Not sure if we need to be as verbose (for example, the table could be xi.spells.enhancing, and the function wouldn't really need to have enhancing in its name as well.